app/components/Form/SubmitButtons.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+import Button from '@button-inc/bcgov-theme/Button';
+import LoadingSpinner from 'components/LoadingSpinner';
+
+const StyledFlex = styled('div')`
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width: 600px) {
+    flex-direction: row;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  margin: 24px 0;
+  white-space: nowrap;
+  min-width: 178px;
+
+  @media (min-width: 600px) {
+    margin: 0 24px;
+  }
+`;
+
+const StyledToast = styled('div')`
+  background-color: ${(props) => props.theme.color.successGreen};
+  border-radius: 4px;
+  padding: 12px 24px;
+  color: #ffffff;
+  font-size: 16px;
+  transition: all ease-in-out 0.2s 0.2s;
+  text-align: center;
+
+  & a {
+    color: #ffffff;
+  }
+`;
+
+type Props = {
+  disabled: boolean;
+  formData: any;
+  isSubmitPage: boolean;
+  isUpdating: boolean;
+  saveAsDraft: boolean;
+  saveForm: any;
+  status: string;
+};
+
+const SubmitButtons = ({
+  disabled,
+  formData,
+  isSubmitPage,
+  isUpdating,
+  saveAsDraft,
+  saveForm,
```

Comment:
Thoughts on calling this `onSubmit`? This would be in line with idiomatic variable naming conventions.

Reply:
`onSave`, or `onClick` would be valid options too. Props that are functions called on an action tend to be called `onSomething`.

If `onSubmit` is confusing, is the name of the component also confusing? Should it be `SaveButtons`?

Going further, if we are to adopt idiomatic naming conventions, the `saveForm` function would be better defined as `handleSave`, or `handleSaveForm` (but maybe that's a change for another PR)

Reply:
`SubmitButtons` is fine since the primary use for this is to submit the form. This feature is on one page and makes sense to have in the same component. `onSave` or `onClick` is confusing as well because it is not clear that what it is doing, remember that we have the main submit button for the entire form in this component which is controlled by the `handleSubmit` in the parent.